doc/cookbook.rst

```diff
@@ -1,15 +1,58 @@
+.. highlight:: rest
 
-=====
+==============
 PYPIT Cookbook
-=====
+==============
 
-1. Install PYPIT
+This document gives an overview on
+how to run PYPIT, i.e. with less detail.
+Notes on :doc:`installing` are found elsewhere.
 
-- Instructions to install at: https://github.com/PYPIT/PYPIT/blob/master/doc/installing.rst
-- The installation requires to define the environment variable GSL_PATH and linetools (included in astropy)
-- Make sure that you are using linetools v0.2 or newer
+The following outlines the standard steps for running
+PYPIT on a batch of data.  There are alternate ways to
+run these steps, but non-experts should adhere to the
+following approach.
 
-2. Prepare the data
+Outline
++++++++
+
+Here is the basic outline of the work flow:
+
+1. Prepare your data
+
+    - Identify folder(s) with raw images
+    - The raw images can be gzip compressed
+
+2. Generate the .setup file (see :doc:`setup`)
+
+    - Run the pypit_setup script
+    - Inspect the .setup file to confirm the instrument configurations
+    - Inspect the .group file to confirm calibrations exist
+    - Modify the data and spect blocks in .pypit file, as needed
```

Comment:
I'm a bit iffy on this line because generating the .pypit file is listed as the next step in this work flow--maybe 2. and 3. should be merged, but I think it's ok to leave it for now. Perhaps after using these scripts routinely myself, I can come back and edit some things if necessary.